Wikipedia/Code/NotificationsCenterCell.swift

```diff
@@ -0,0 +1,342 @@
+import UIKit
+
+protocol NotificationsCenterCellDelegate: AnyObject {
+	func userDidTapSecondaryActionForCellIdentifier(id: String)
+}
+
+final class NotificationsCenterCell: UICollectionViewCell {
```

Comment:
@staykids One more thing - the RTL scheme has things looking slightly off.  The date label needs to be right aligned and I think the truncation is on the wrong side:


![Screen Shot 2021-09-29 at 9 47 23 AM](https://user-images.githubusercontent.com/3620196/135293025-2cb168df-17c2-4280-aa01-2586ccfac68b.png)

Reply:
> @staykids One more thing - the RTL scheme has things looking slightly off. The date label needs to be right aligned 

Good catch – updated the time label to change its text alignment depending on the user interface layout direction. 

> and I think the truncation is on the wrong side:

Regarding the truncation, my thinking here could be wrong given the _interesting_ ways we support LTR/RTL, but copy and pasting some RTL language text into the labels, it looks like the truncation is already properly happening on the left side.

![RTL](https://user-images.githubusercontent.com/5652327/135325881-5fdd0ed3-6a89-4362-b9a6-6c4917adc33e.png)

Reply:
@staykids aah nice, I didn't actually try pasting in RTL text. Thanks for confirming!